port/unix/omrmmap.c

```diff
@@ -337,6 +456,15 @@ omrmmap_get_region_granularity(struct OMRPortLibrary *portLibrary, void *address
 	return protect_region_granularity(portLibrary, address);
 }
 
+#if defined(J9ZOS390)
+#pragma linkage (PGSERRM,OS)
+#pragma map(Pgser_Release,"PGSERRM")
+#if defined(OMR_ENV_DATA64)
+#pragma linkage(omrdiscard_data,OS_NOSTACK)
```

Comment:
Please be consistent: either the space before `(` in `pragma linkage` is appropriate or it's not.